src/index.html

```diff
@@ -11,6 +11,33 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+
+    <div class="container">
+      <div class="search-bar__big"
+        data-qa="big"
+        >
+        <span class="icon">
+          <img src="images/Search.svg" alt="svg">
+        </span>
+        <input
+         type="text"
+         data-qa="keypress"
+         value="Try “Los Angeles”"
+         >
```

Comment:
```suggestion
        <input
         type="text"
         data-qa="keypress"
         value="Try “Los Angeles”"
       >
```
![Знімок екрана 2023-10-23 о 16 09 10](https://github.com/mate-academy/layout_search-bar-airbnb/assets/106310796/b5ee7deb-14e0-4ac2-b0e2-f084e6611d74)